### ISSUE TITLE
Can't enable laptop monitor while external monitor is enabled, since kernel 6.8

### ISSUE THREAD

**Issue by birgersp** [pop-os/linux#316]
Laptop model: Asus ROG G15, G513QM-HF144T

After upgrading to kernel 6.8, my monitor setup is "broken".

It seems that I can't have the laptop display enabled, while also having enabled any of my (two) external monitors.

Specifically, I try to enable the laptop display (display 1) in the Displays settings. If I hit "Apply" at this stage (image);

![image](https://github.com/pop-os/linux/assets/5260237/5124c008-e5f3-4c66-a3e9-271b8d98b9b6)

What happens is that the screen flashes for a couple of seconds, then display no 1 is "automatically" disabled again.

![image](https://github.com/pop-os/linux/assets/5260237/77206aef-eaa7-4f61-9950-419234d4b556)

So in short, I can't enable it, because it just gets automatically disabled again when I click "Apply".

(I'm on 6.6.10 again now and that works like a charm)

**Comment by leviport** [comment]
I believe this is a duplicate of https://github.com/pop-os/pop/issues/3257 (and possibly related to https://github.com/pop-os/linux/issues/304)

**Comment by birgersp** [comment]
> I believe this is a duplicate of [pop-os/pop#3257](https://github.com/pop-os/pop/issues/3257) (and possibly related to #304)

In my opinion, the issues are different. Yes they're both concerning the Displays settings dialog, but this issue is about not being able to enable the laptop monitor when external monitor is enabled.

Please re-open this issue 🙏

**Comment by leviport** [comment]
Pretty sure this is just a different symptom of the same bug, but sure, I guess we can leave this open.
